config/tasklists/learn-to-drive-a-car.json

```diff
@@ -0,0 +1,154 @@
+            },
+            {
+              "href": "/government/publications/car-show-me-tell-me-vehicle-safety-questions",
+              "text": "Practise vehicle safety questions"
+            }
+          ]
+        },
+        {
+          "title": "Prepare for your theory test",
+          "panel_links": [
+            {
+              "href": "/theory-test/revision-and-practice",
+              "text": "Theory test revision and practice"
+            },
+            {
+              "href": "/take-practice-theory-test",
+              "text": "Take a practice theory test"
+            },
+            {
+              "href": "https://www.safedrivingforlife.info/shop/product/official-dvsa-theory-test-kit-app-app",
+              "text": "Theory and hazard perception test app"
+            }
+          ]
+        }
+      ],
+      [
+        {
+          "title": "Book and manage your theory test",
+          "panel_descriptions": ["You need a provisional driving licence to book your theory test."],
+          "panel_links": [
+            {
+              "href": "/book-theory-test",
+              "text": "Book your theory test"
+            },
+            {
+              "href": "/theory-test/what-to-take",
+              "text": "What to take to your test"
+            },
+            {
+              "href": "/change-theory-test",
+              "text": "Change your theory test appointment"
+            },
+            {
+              "href": "/check-theory-test",
+              "text": "Check your theory test appointment details"
+            },
+            {
+              "href": "/cancel-theory-test",
+              "text": "Cancel your theory test"
+            }
+          ]
+        }
+      ],
+      [
+        {
+          "title": "Book and manage your driving test",
+          "panel_descriptions": ["You must pass your theory test before you can book your driving test."],
+          "panel_links": [
+            {
+              "href": "/book-driving-test",
+              "text": "Book your driving test"
+            },
+            {
+              "href": "/driving-test/what-to-take",
+              "text": "What to take to your test"
+            },
+            {
+              "href": "/change-driving-test",
+              "text": "Change your driving test appointment"
+            },
+            {
+              "href": "/check-driving-test",
+              "text": "Check your driving test appointment details"
+            },
+            {
+              "href": "/cancel-driving-test",
+              "text": "Cancel your driving test"
+            }
+          ]
+        }
+      ],
+      [
+        {
+          "title": "When you pass",
+          "panel_descriptions": ["You can start driving as soon as you pass your driving test."],
+          "panel_links": [
+            {
+              "href": "/pass-plus",
+              "text": "Pass Plus"
```

Comment:
This text has updated to "Find out about Pass Plus training courses"
See https://docs.google.com/spreadsheets/d/1A6pWxHloiMvExtiQNmL-8m-ruXQFRluNObK28uhU1jI/edit#gid=1049909139